corehq/apps/reports/templates/reports/bootstrap3/base_template.html

```diff
@@ -5,7 +5,7 @@
 {% block js %}{{ block.super }}
     {% compress js %}
     <script src="{% new_static 'hqwebapp/js/ajax_csrf_setup.js' %}"></script>
-    <script src="{% new_static 'style/js/bootstrap2/main.js' %}"></script>
+    <script src="{% new_static 'style/js/bootstrap3/main.js' %}"></script>
```

Comment:
i don't know what else is implied by this change, but confirming it's not something that we think needs any QA?